.github/workflows/run_linux_smoke_tests.yml

```diff
@@ -26,8 +26,39 @@ jobs:
     name: Run Linux Smoke Tests
     runs-on: ubuntu-latest
 
+    env:
+      test_results_path: tests\TestResults
+      integration_tests_shared_project: ${{ github.workspace }}/tests/Agent/IntegrationTests/Shared
+      NR_DOTNET_TEST_SAVE_WORKING_DIRECTORY: 1
+      # Make this variable true to enable extra data-gathering and logging to help troubleshoot test failures, at the cost of additional time and resources
+      enhanced_logging: false
+  
+  
     steps:
+      - name: Docker Setup QEMU
+        uses: docker/setup-qemu-action@v2.2.0
```

Comment:
Do we need to pin this to a specific commit?

Reply:
Good catch, yeah we'll need to pin it or the security scanner will get mad...